### DISCUSSION TITLE
検索結果のリスト表示のコントラストが低い

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#296] [要望・アイデア・不具合]

**Comment by tanaka-asami415** [comment]
@tsubolabo 
ご意見ありがとうございます🙏
確かに読みづらいかもしれないので検討いたします🙇

**Reply by tanaka-asami415** [reply]
@tsubolabo 
ありがとうございます！
多くの方に情報が見やすくなるよう、バランスよく考慮したいと思います🙏
1ユーザーさんだけのご意見を取り入れるということはなく、
多くのユーザーさんの体験がよくなるよう改善をしております！

**Reply by tanaka-asami415** [reply]
@tsubolabo 
お待たせしました🙇
検索結果のコントラストを変更したのですが、いかがでしょうか？
以前より見やすくなっていると幸いです！
